client/src/components/RelatedItems/YourOutfit.jsx

```diff
@@ -1,7 +1,43 @@
-import { React } from "react";
+import React, { useContext } from 'react';
+import {
+  FaPlusSquare,
+  FaArrowAltCircleLeft,
+  FaArrowAltCircleRight,
+} from 'react-icons/fa';
 
-function YourOutfit( props ) {
-  return <div> Your Outfit </div>;
+import ProductContext from '../Context';
+
+function YourOutfit({ setYourOutfitId }) {
+  const { productId, yourOutfit, setProductId } = useContext(ProductContext);
+
+  return (
+    <div>
+      <section className="yo-slider">
+        <div className="yo-button-container">
+          <FaPlusSquare
+            className="yo-additem"
+            onClick={() => setYourOutfitId(productId)}
+          />
+          <h3>Add to Your Outfit!</h3>
+        </div>
+        {yourOutfit.length > 0 &&
+          yourOutfit.map((product, index) => (
+            <div key={index}>
+              <div className="yo-container">
+                <img
+                  className="yo-image"
+                  src={product.results[0].photos[0].thumbnail_url}
+                  alt="clothing"
+                  key={product.product_id}
+                  onClick={() => setProductId(product.product_id)}
```

Comment:
Should the items in the Your Outfit section affect the global productId?

I thought I read that the only interactions for Your Outfit should be the ability to add or remove items from the outfit list.

Reply:
I know...I sort of left it as a place holder and was enjoying the click functionality!